.gitignore

```diff
@@ -74,11 +74,6 @@ config/deployer.json
 config/comparison.json
 config/metricswatcher.json
 
-# binaries
-ltagent
```

Comment:
You can use `/ltagent` and it will exactly match only ltagent under the root dir. Rest of the changes are fine.

Reply:
Done in https://github.com/mattermost/mattermost-load-test-ng/pull/508/commits/8a5ff5a2ac83f62516ee67ed4cd4d2c396bece27